web/src/app/pages/all-pushes/all-pushes.component.ts

```diff
@@ -1,15 +1,37 @@
 import { Component, OnInit } from '@angular/core';
```

Comment:
OnInit is no longer used. 

Same as for my-pushes page.

Reply:
I'm going to fix the my-pushes in a separate PR.